core/src/main/java/org/mskcc/cbio/portal/util/StructuralVariantUtil.java

```diff
@@ -0,0 +1,162 @@
+
+    // Column names structural variant file
+    public static final String VARIANT_CLASS = "Class";
+    public static final String SITE2_DESCRIPTION = "Site2_Description";
+    public static final String SITE1_EXON = "Site1_Exon";
+    public static final String SITE1_ENTREZ_GENE_ID = "Site1_Entrez_Gene_Id";
+    public static final String DRIVER_FILTER = "cbp_driver";
+    public static final String SITE2_HUGO_SYMBOL = "Site2_Hugo_Symbol";
+    public static final String NORMAL_SPLIT_READ_COUNT = "Normal_Split_Read_Count";
+    public static final String SITE2_EFFECT_ON_FRAME = "Site2_Effect_On_Frame";
+    public static final String SAMPLE_ID = "Sample_ID";
+    public static final String DRIVER_TIERS_FILTER = "cbp_driver_tiers";
+    public static final String SITE2_POSITION = "Site2_Position";
+    public static final String COMMENTS = "Comments";
+    public static final String DRIVER_TIERS_FILTER_ANNOTATION = "cbp_driver_tiers_annotation";
+    public static final String TUMOR_READ_COUNT = "Tumor_Read_Count";
+    public static final String SITE1_HUGO_SYMBOL = "Site1_Hugo_Symbol";
+    public static final String SITE1_POSITION = "Site1_Position";
+    public static final String NORMAL_PAIRED_END_READ_COUNT = "Normal_Paired_End_Read_Count";
+    public static final String NORMAL_READ_COUNT = "Normal_Read_Count";
+    public static final String SITE1_DESCRIPTION = "Site1_Description";
+    public static final String NCBI_BUILD = "NCBI_Build";
+    public static final String ANNOTATION = "Annotation";
+    public static final String CENTER = "Center";
+    public static final String RNA_SUPPORT = "RNA_Support";
+    public static final String DRIVER_FILTER_ANNOTATION = "cbp_driver_annotation";
+    public static final String SITE2_CHROMOSOME = "Site2_Chromosome";
+    public static final String TUMOR_VARIANT_COUNT = "Tumor_Variant_Count";
+    public static final String DNA_SUPPORT = "DNA_Support";
+    public static final String BREAKPOINT_TYPE = "Breakpoint_Type";
+    public static final String SITE2_ENTREZ_GENE_ID = "Site2_Entrez_Gene_Id";
+    public static final String EVENT_INFO = "Event_Info";
+    public static final String CONNECTION_TYPE = "Connection_Type";
+    public static final String TUMOR_PAIRED_END_READ_COUNT = "Tumor_Paired_End_Read_Count";
+    public static final String EXTERNAL_ANNOTATION = "External_Annotation";
+    public static final String TUMOR_SPLIT_READ_COUNT = "Tumor_Split_Read_Count";
+    public static final String SITE1_CHROMOSOME = "Site1_Chromosome";
+    public static final String SITE2_EXON = "Site2_Exon";
+    public static final String LENGTH = "Length";
+    public static final String SITE1_ENSEMBL_TRANSCRIPT_ID = "Site1_Ensembl_Transcript_Id";
+    public static final String SITE2_ENSEMBL_TRANSCRIPT_ID = "Site2_Ensembl_Transcript_Id";
+    public static final String NORMAL_VARIANT_COUNT = "Normal_Variant_Count";
+
+    public StructuralVariantUtil(){}
```

Comment:
If an instance is created with the default constructor, it seems like it will be non-functional because the column indices will not have been set and there is no other mechanism to set them aside from the other constructor. Perhaps this should be declared private so that it prevents use of the default constructor.

Reply:
@sheridancbio I needed the default constructor for unit testing